tdrs-backend/docker-compose.yml

```diff
@@ -69,6 +69,52 @@ services:
     volumes:
       - elastic_data:/usr/share/elasticsearch/data
 
+  grafana:
+    restart: always
+    image: grafana/grafana:11.2.0
+    ports:
+      - 9400:9400
+    volumes:
+      - ./plg/grafana/datasources.yml:/etc/grafana/provisioning/datasources/default.yml
+      - ./plg/grafana/dashboards/provider.yml:/etc/grafana/provisioning/dashboards/default.yml
+      - ./plg/grafana/dashboards:/var/lib/grafana/provisioning/dashboards
+      - ./plg/grafana/custom.ini:/etc/grafana/grafana.ini
+      - grafana_data:/var/lib/grafana
+
+  prometheus:
+    restart: always
+    image: prom/prometheus:v2.54.1
+    ports:
+      - 9090:9090
+    volumes:
+      - ./plg/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./plg/prometheus/django_rules.yml:/etc/prometheus/prom_django_rules.yml
+      - prometheus_data:/prometheus
+    depends_on:
+      - web
+      - celery-exporter
+      - postgres-exporter
+
+  celery-exporter:
+    restart: always
+    image: ovalmoney/celery-exporter:1.5.1
```

Comment:
This is an interesting exporter, really like the way it exports the metrics. How are we planning to implement this in cloud.gov? dockerized? I guess it should sit with redis?